### DISCUSSION TITLE
Import into new slack workspace

### DISCUSSION THREAD

**Discussion by BwL1289** [rusq/slackdump#165] [Q&A] [answered]
Awesome tool. I've exported to both `json` and `txt` and am now trying to reimport conversations into a new slack workspace but getting various errors (like, format not supported for `txt`). I know importing via json is not supported. For reference, I am trying to follow these guidelines: https://slack.com/help/articles/360035354694-Move-data-to-Slack-using-a-CSV-or-text-file.

I believe I have to reformat the `txt` file to make it importable. Apologies in advance if there's an easy way to do this and I'm missing something simple.

I've spoken with slack support and they recommended reaching out to the slack dump community.

Thanks so much in advance.

**Comment by rusq** [comment, marked as answer]
Hey @BwL1289 , thanks for your question.

It's the first time I'm hearing about the CSV/Text import into Slack (and obviously slackdump does not support it for this reason).  The text format was designed for convenience of reading/archiving conversations.

I'll have a read through the docs and see what I can do, seems like a useful feature to have.

I'll create an issue from this discussion no to lose the visibility.

**Reply by rusq** [reply]
https://github.com/rusq/slackdump/issues/166

**Reply by BwL1289** [reply]
Thank you so much, @rusq!
